libs/auth-api-lib/src/lib/entities/models/delegation.model.ts

```diff
@@ -48,6 +48,40 @@ export class Delegation extends Model<Delegation> {
   })
   toName!: string
 
+  get validTo(): Date | null | undefined {
+    // 1. Find a value with null as validTo. Null means that delegation scope set valid not to a specific time period
+    const withNullValue = this.delegationScopes?.find((x) => x.validTo === null)
+    if (withNullValue) {
+      return null
+    }
+
+    // 2. Find the max value
+    try {
```

Comment:
why the try/catch?

Reply:
I'm mapping an array that could return null. I'm not sure if that will produce an error or not. I could map it after I see it has some elements
const arrDates = this.delegationScopes
        ?.filter((x) => x.validTo !== null && x.validTo !== undefined)
        .map((x) => x.validTo)